### DISCUSSION TITLE
Data Base URL not in config file on Firebase

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#123] [Help Me!] [answered]
I went through the Firebase setup per video, but when the copy/pasting over the keys, I there wasn't an option for the data base URL. How can I add this?

No other issue tickets

https://firebase.google.com/docs/dynamic-links/custom-domains
https://stackoverflow.com/questions/40168564/where-can-i-find-my-firebase-reference-url-in-firebase-account

**Comment by drteresavasquez** [comment, marked as answer]
Make sure that you create your RealTime Database first then create an App on the project.

**Comment by drteresavasquez** [comment]
@danicrosby I also deleted your keys from your ticket since we want to keep those private because hackers 😄

**Comment by danicrosby** [comment]
Weirdly I was able to set the database up after the fact and it gave me a URL. ?? I'm back on track.  I'll go with it and see what happens.

**Comment by danicrosby** [comment]
Update: I have been able to process as planned, imported authors and books from my finder data with success.
